src/inference/hmc.jl

```diff
@@ -497,8 +422,7 @@ function steps!(
     vi,
     samples;
     rng::AbstractRNG=GLOBAL_RNG,
-    verbose::Bool=true,
-    progress::Bool=false
```

Comment:
I believe this is a merge issue, we should not drop progress. Same for all places